app/apollo/resolvers/channel.js

```diff
@@ -961,23 +989,22 @@ const channelResolvers = {
 
         // Get the Version (avoid using the deprecated/ignored `versions` attribute on the channel)
         const deployableVersionObj = await models.DeployableVersion.findOne({ org_id, uuid });
-        if(!deployableVersionObj){
-          // If unable to find the Version, throw an error
-          throw new NotFoundError(context.req.t('Version uuid "{{version_uuid}}" not found.', {'version_uuid':uuid}), context);
-        }
 
-        // Get the Channel for the Version
         const channel = await models.Channel.findOne({ uuid: deployableVersionObj.channel_id, org_id });
-        if(!channel){
+        if (!channel) {
           // If unable to find the Channel then cannot verify authorization.  Assume deletion allowed rather than throwing an error (else version would be permanently undeletable).
           logger.warn({ver_uuid: uuid, ver_name: deployableVersionObj.name}, `${queryName} channel for version ${uuid} not found, authorization assumed.`);
         }
         else {
-          // Channel is found, verify authorization on the Channel
           await validAuth(me, org_id, ACTIONS.MANAGEVERSION, TYPES.CHANNEL, queryName, context, [channel.uuid, channel.name]);
+          logger.info({req_id, user, org_id, uuid}, `${queryName} validating - authorized`);
+        }
+
+        if(!deployableVersionObj){
```

Comment:
The original location for checking if deployableVersionObj was found (right after querying the DB for it) must be kept -- if the version was _not_ found, cannot get the channel_id from it to check auth.

Reply:
fixed

Reply:
Unresolving conversation, not fixed?

Reply:
This is outdated, it is fixed.

Reply:
I think the fault was on my end -- refreshing a diff between specific commits rather than getting the overall diff for the PR fresh.
Resolved.